locale/core.yml

```diff
@@ -320,6 +320,11 @@ core:
       username_placeholder: => core.ref.username
       welcome_text: "Welcome, {username}!"
 
+    # These translations are used to show user an alert if the account didn't verfied
+    resend_confirmation:
+      alert_message: You need to confirm your email
```

Comment:
A suggestion to match the style of the sentence to the rest of the UI:

      alert_message: You need to confirm your email _address._